spec/system/page_objects/pages/topic.rb

```diff
@@ -114,6 +115,32 @@ def fill_in_composer_title(title)
         @composer_component.fill_title(title)
       end
 
+      def fast_edit_button
+        find(".quote-button .quote-edit-label")
+      end
+
+      def click_fast_edit_button
+        find(".quote-button .quote-edit-label").click
+      end
+
+      def fast_edit_input
```

Comment:
This should just call `@fast_edit_component.fast_edit_input`, maybe this is just leftover?

Reply:
Good catch, will update this to use the component